src/commands/synthetics/__tests__/cli.test.ts

```diff
@@ -3,6 +3,8 @@ import {Cli} from 'clipanion/lib/advanced'
 import {createCommand} from '../../../helpers/__tests__/fixtures'
 import * as ciUtils from '../../../helpers/utils'
 
+import {DATADOG_API_KEY_REG_EXP} from '../../lambda/constants'
```

Comment:
Let's not import from `lambda`, and move their constant to the general `src/constants.ts` file. Their team will be notified for a review automatically

Reply:
oups, it's actually not being used haha, gotta love auto import 😅